lib/codec_choose.c

```diff
@@ -49,6 +49,9 @@
 #ifndef bit_SSE42
 #define bit_SSE42 (1 << 20)
 #endif
+#ifndef bit_AVX
+#define bit_AVX (1 << 28)
+#endif
 
 #define bit_XSAVE_XRSTORE (1 << 27)
```

Comment:
I completely missed bit 28, I must be going blind. Wikipedia informs us that bit 5 in `EAX=7` should be the `avx2` bit. I believe the current code was based on [this](https://software.intel.com/en-us/articles/how-to-detect-new-instruction-support-in-the-4th-generation-intel-core-processor-family) Intel snippet or a derivative.